content/blogs/2023-09-18-kestra-terraform-partnership.md

```diff
@@ -0,0 +1,57 @@
+
+>“Integrating Kestra with Terraform enables a cohesive approach to automating your workflows. It's not just about tooling; it's about  streamlining operations in a way that resonates with core DevOps principles.”
+
+*Ludovic Dehon CTO @ Kestra*
+
+---
+
+## Why Terraform and Kestra? A Technologic Alignment ##
+
+Our partnership with Terraform goes beyond just working together; it's about synchronizing our strengths. Both of our technologies aim to make your operational tasks less complicated. With this partnership, you can easily configure Kestra workflows and tasks within your Terraform settings. This coordination makes scaling straightforward and keeps your operational changes well-documented through a code-based approach.
+
+--- 
+
+## Elevating Your Data Workflow With Kestra's Terraform Provider ##
+
+![Terraform Kestra](/blogs/2023-09-18-kestra-terraform-partnership/terraform-kestra.png)
+
+Our Terraform provider is a significant step in aligning data orchestration with infrastructure-as-code best practices. With this integration, configuring Kestra workflows and tasks directly within your Terraform scripts is a seamless experience. It's an integrated approach to streamline and automate all your flows.
+
+
+### Breaking the Production Barrier: The Kestra and Terraform Solution ###
+
+Moving data pipelines from development to production comes with its own set of challenges. Be it environment setups, database schema access, or cloud resource permissions, the list seems endless. The alliance between Kestra and Terraform aims to bridge this gap by integrating open-source tools that make this transition easier for developers. Learn more about [Everything as Code for Data Infrastructure solution](https://levelup.gitconnected.com/programmable-data-infrastructure-is-finally-within-reach-83fb8e6392ac).
+
+
+### Supercharge Your CI/CD with Terraform & Kestra ###
+Bring Kestra into your CI/CD pipelines using tools like GitHub Actions. By doing so you can automate your infrastructure provisioning but also deploy your Kestra flows in the CI/CD model. With a GitHub Actions workflow, you can also validate and deploy flows directly into different namespaces ensuring that your data workflows are as agile and adaptable as your infrastructure.
+<br>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=jXSj49xwStR5Wqk8&amp;list=PLEK3H8YwZn1olYeQ7qbGdY5XQySKCNLUr" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>
+
+### Unlocking Data Efficiency: 5 Million Executions and 4,000+ Flows with Kestra and Terraform ###
```

Comment:
```suggestion
### 10x Your Workflow Automation and Software Delivery
```

those numbers are not encouraging - executions are not deployed from terraform, and 4k flows seems like not much, I'd cut it